applications/nrf5340_audio/src/modules/nrf5340_audio_dk.c

```diff
@@ -1,30 +1,21 @@
 /*
```

Comment:
Let's have a brief talk on the location where this file is stored.

Reply:
Moved to utils